### PR TITLE
Update nephio repo readme to be release neutral

### PR DESCRIPTION
**Pull request by liamfallon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,18 +5,16 @@ deploying and managing highly distributed, interconnected workloads such as 5G
 Network Functions, and the underlying infrastructure on which those workloads
 depend.
 
-The R1 Release is now available! To get started with Nephio, view the [docs repo](https://github.com/nephio-project/docs) 
-which contains the [R1 Demo installation instructions](https://github.com/nephio-project/docs/blob/main/install-guide/README.md) 
-and [Quick start exercises](https://github.com/nephio-project/docs/blob/main/user-guide/exercises.md). 
-An announcement from The Linux Foundation regarding the release can be found 
-[here](https://www.linuxfoundation.org/press/nephio-community-gains-momentum-with-release-1-to-simplify-cloud-native-network-automation).
+To get started with Nephio, view the [docs repo](https://github.com/nephio-project/docs)
```
Comment (electrocucaracha) [inline README.md:8]:
```suggestion
To get started with Nephio, view the [documentation website](https://docs.nephio.org/docs/) 
```

**Comment by tliron** [comment]
> What about to start using the official website

For now, docs.nephio.org will track main, so this is a good idea. In the future as part of release we may want to match a tagged README with a versioned doc, e.g. something like `docs.nephio.org/v3.0.0`. It would be hard to do manually, we will need to brainstorm a simpler solution.

**Comment by liamfallon** [comment]
@electrocucaracha , @tliron , @efiacor , @henderiw updated following comments.

**Pull request merged**